build_taglib.bat

```diff
@@ -32,50 +32,25 @@ mkdir %PLATFORM%
 cd %PLATFORM%
 REM NOTE(pegasus): generated solution with
 
```

Comment:
taglib doesn't rely on Qt so I don't think the `CMAKE_PREFIX_PATH` change is necessary?

Reply:
No, we can remove this.